concepticondata/conceptlists/Capell-1956-558.tsv

```diff
@@ -0,0 +1,559 @@
+Capell-1956-558-234	1448	HOLD	234	hold	Capell1956[59]
+Capell-1956-558-235	1667	HOLE	235	hole	Capell1956[59]
+Capell-1956-558-236	942	HONEY	236	honey	Capell1956[59]
+Capell-1956-558-237	665	BEE	237	honey-bee	Capell1956[60]
+Capell-1956-558-238	1074	FISHHOOK	238	hook (for fishing)	Capell1956[60]
+Capell-1956-558-239	1286	HOT	239	hot	Capell1956[60]
+Capell-1956-558-240	1533	BE HUNGRY	240	hungry	Capell1956[60]
+Capell-1956-558-241	2411	HURT (SOMEBODY)	241	hurt	Capell1956[61]
+Capell-1956-558-242	1200	HUSBAND	242	husband	Capell1956[61]
+Capell-1956-558-243	1678	HUT	243	hut	Capell1956[61]
+Capell-1956-558-244	1209	I	244	I	Capell1956[62]
+Capell-1956-558-245	621	IRON	245	iron	Capell1956[62]
+Capell-1956-558-246	622	ISLAND	246	island	Capell1956[62]
+Capell-1956-558-247	1578	IGUANA	247	iguana	Capell1956[62]
+Capell-1956-558-248			248	jabiru	Capell1956[63]
+Capell-1956-558-249	2821	JEALOUS	249	jealous	Capell1956[63]
+Capell-1956-558-250	1829	JUMP	250	jump	Capell1956[63]
+Capell-1956-558-251	164	KANGAROO	251	kangaroo	Capell1956[63]
+Capell-1956-558-252			252	kangaroo-big	Capell1956[64]
+Capell-1956-558-253			253	kangaroo-small	Capell1956[64]
+Capell-1956-558-254	1135	KICK	254	kick	Capell1956[64]
+Capell-1956-558-255	489	KIDNEY	255	kidney	Capell1956[64]
+Capell-1956-558-256	1417	KILL	256	kill	Capell1956[65]
+Capell-1956-558-257	2444	KINGFISHER	257	kingfisher	Capell1956[65]
+Capell-1956-558-258	1371	KNEE	258	knee	Capell1956[65]
+Capell-1956-558-259	1352	KNIFE	259	knife	Capell1956[65]
+Capell-1956-558-260			260	knick up	Capell1956[66]
+Capell-1956-558-261	3626	KNOW	261	know	Capell1956[66]
+Capell-1956-558-262			262	kulamin	Capell1956[66]
+Capell-1956-558-263	2444	KINGFISHER	263	kookaburra	Capell1956[66]
+Capell-1956-558-264	320	LAME	264	lame	Capell1956[67]
+Capell-1956-558-265	1307	LANGUAGE	265	language	Capell1956[67]
+Capell-1956-558-266	1355	LAUGH	266	laugh	Capell1956[67]
+Capell-1956-558-267	628	LEAF	267	leaf	Capell1956[67]
+Capell-1956-558-268			268	leach	Capell1956[68]
+Capell-1956-558-269	244	LEFT	269	left	Capell1956[68]
+Capell-1956-558-270	1297	LEG	270	leg	Capell1956[68]
+Capell-1956-558-271	215	LIE DOWN	271	lie (down)	Capell1956[68]
+Capell-1956-558-272	2826	LIE (FALSEHOOD)	272	lie (tell a)	Capell1956[69]
+Capell-1956-558-273	210	LIFE	273	life	Capell1956[69]
+Capell-1956-558-274	501	LIFT	274	lift	Capell1956[69]
+Capell-1956-558-275	737	LIGHT (IGNITE)	275	light (a fire)	Capell1956[69]
+Capell-1956-558-276	1052	LIGHT (WEIGHT)	276	light (in weight)	Capell1956[70]
+Capell-1956-558-277	1654	LIGHTNING	277	lightning	Capell1956[70]
+Capell-1956-558-278			278	lily	Capell1956[70]
+Capell-1956-558-279			279	lily-root	Capell1956[70]
+Capell-1956-558-280	478	LIP	280	lip	Capell1956[71]
+Capell-1956-558-281	1608	LISTEN	281	listen	Capell1956[71]
+Capell-1956-558-282	1246	SMALL	282	little	Capell1956[71]
+Capell-1956-558-283			283	little while	Capell1956[71]
+Capell-1956-558-284	1422	BE ALIVE	284	live	Capell1956[72]
```

Comment:
This should probably be [2309 LIVE](https://concepticon.clld.org/parameters/2309)